recipes/r-wordcloud/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '2.5' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-wordcloud
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: wordcloud_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/wordcloud_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/wordcloud/wordcloud_{{ version }}.tar.gz
+  sha256: 95e173192daa90065ffe617094528b5c0bf7fbd9a8fd40da68910788ea6c58aa
+
+build:
+  number: 0
+  skip: true  # [win32]
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-slam
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - r-rcolorbrewer
+    - r-rcpp
+    - r-slam
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('wordcloud')"  # [not win]
+    - "\"%R%\" -e \"library('wordcloud')\""  # [win]
+
+about:
+  home: http://blog.fellstat.com/?cat=11 http://www.fellstat.com http://research.cens.ucla.edu/
```

Comment:
two URLs?

Reply:
Is two the limit? I just listed all that are stated on the CRAN page.